cli/tauri.js/src/types/config.ts

```diff
@@ -223,6 +223,7 @@ export interface TauriConfig {
        * whether we should use the embedded-server or the no-server mode
        */
       active?: boolean
+      port?: string
```

Comment:
Do we really want to use a `string` here, or would a number be more appropriate? @lucasfernog

Reply:
at the rust runner code, it do compare if port == "random", unless changing that too..

Reply:
to be a number the rust code needs changes

Reply:
its not a big deal at all